src/llmcompressor/transformers/tracing/glm/LICENSE

```diff
@@ -0,0 +1,84 @@
+The glm-4-9b License
```

Comment:
Could you check if we need to add this to the manifest file? or will it get picked up in our wheels automatically?

Reply:
I would not have caught that, thank you!

It is not included by default. After I added it to the manifest.in, you can verify that it is picked up as so:
```bash
python setup.py sdist
tar -tzf dist/llmcompressor-dev-0.3.1.dev0.tar.gz | grep LICENSE
```